src/main/java/it/gov/pagopa/debtposition/controller/pd/crud/api/impl/DebtPositionController.java

```diff
@@ -56,7 +61,10 @@ public ResponseEntity<PaymentPositionModel> createDebtPosition(String organizati
         PaymentPosition createdDebtPos = paymentPositionService.create(debtPosition, organizationFiscalCode, toPublish);
 
         if (null != createdDebtPos) {
-            return new ResponseEntity<>(modelMapper.map(createdDebtPos, PaymentPositionModel.class), HttpStatus.CREATED);
+        	PaymentPositionModel paymentPosition = modelMapper.map(createdDebtPos, PaymentPositionModel.class);
+        	//PAGOPA-1155: add nav info to PO
+        	paymentPosition.getPaymentOption().forEach(po -> po.setNav(auxDigit+po.getIuv()));
```

Comment:
Can the operation "auxDigit+po.getIuv()" (and so the injection of the auxDigit value from env parameter with @Value annotation) be included [in this class](https://github.com/pagopa/pagopa-debt-position/blob/main/src/main/java/it/gov/pagopa/debtposition/util/CommonUtil.java) as utility method? Surely can avoid code duplication and make code more clean.

Reply:
from a comparison with @jacopocarlini it was decided to leave the current enhancement of the NAV